src/pipes/search.ts

```diff
@@ -0,0 +1,18 @@
+import { Pipe } from '@angular/core';
+
+@Pipe({
+  name: 'search'
+})
+export class SearchPipe {
+  transform(list, pattern = '', deepKey = '') {
+    if (!list) return;
+
+    return list.filter((item) => {
+      const value = deepKey.split('.').reduce((value, key) => {
```

Comment:
That's what I meant :) `lodash` publishes each one of it's utility functions as a standalone module, for example: https://www.npmjs.com/package/lodash.isstring